modules/core/src/main/java/org/apache/ignite/internal/processors/cache/CacheMetricsImpl.java

```diff
@@ -318,6 +339,12 @@ public CacheMetricsImpl(GridCacheContext<?, ?> cctx, boolean isNear) {
 
             return fut != null && !fut.isDone();
         }, "True if index rebuild is in progress.");
+
+        getHistogram = mreg.histogram("GetHistogram", HISTOGRAM_BUCKETS, "Get duration in nanoseconds.");
```

Comment:
Let's rename this to `GetTime`
And presented `GetTime` to `GetTimeTotal`.

I think other metrics should be renamed with this pattern, also.

Reply:
Done.